CHANGELOG.rst

```diff
@@ -4,6 +4,25 @@ The cloud.terraform collection Release Notes
 
 .. contents:: Topics
 
+v3.1.0
+======
+
+Release Summary
+---------------
+
+This release includes bug fixes and new feature for the ``terraform_state`` inventory plugin.
+
+Minor Changes
+-------------
+
+- Bump version of ansible-lint to minimum 24.7.0 (https://github.com/ansible-collections/cloud.terraform/pull/161).
```

Comment:
Perhaps we can add a manual entry under this one to show that we've updated the `ansible-lint` version to 25.1.2 per #176

Reply:
updated